all.sh

```diff
@@ -8,57 +8,85 @@ import history
 import files
 import programs
 
-NASHCOMPLETE_CMD = (
+var GOBIN = $GOPATH+"/bin"
+var ENZOTOOLS = ($GOBIN+"/ls" $GOBIN+"/echo" $GOBIN+"/cat")
+var UNIXTOOLS = ("sed" "grep")
+var NASHCOMPLETE_CMD = (
 	("kill" $nash_complete_kill)
 	("systemctl" $nash_complete_systemctl)
 )
 
+fn abort(msg) {
+	echo "error: "+$msg
+
+	exit(1)
+}
+
+fn init() {
+	for tool in $ENZOTOOLS {
+		if file_exists($tool) != "0" {
+			abort("Enzo tools not installed: "+$tool+" not found")
```

Comment:
I'm not sure either =(
But this is only an attempt to make this project more reliable.